package com.fiap.techchallenge.fourlanches.domain.exception;

public class CustomerSaveException extends RuntimeException {

    public CustomerSaveException(String message, Throwable cause) {
        super(message, cause);
    }

}
